app/src/main/java/com/sdp/movemeet/map/MiniMapFragment.java

```diff
@@ -41,25 +43,27 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         View view = inflater.inflate(R.layout.fragment_map, container, false);
 
+        currentLocation = LocationFetcher.defaultLocation();
+
         supportMapFragment = (SupportMapFragment) getChildFragmentManager().findFragmentById(R.id.google_map);
 
         fusedLocationProviderClient = LocationServices.getFusedLocationProviderClient(supportMapFragment.getActivity());
-        LocationFetcher.fetchLastLocation(supportMapFragment, fusedLocationProviderClient, this);
+        LocationFetcher.fetchLastLocation(fusedLocationProviderClient, supportMapFragment, this);
 
         return view;
     }
 
     @Override
     public void onMapReady(GoogleMap googleMap) {
         googleMap.setOnMapClickListener(this::onMapClick);
-        currentLocation = LocationFetcher.getLocation();
+        currentLocation = LocationFetcher.currentLocation;
         LatLng location = ((UploadActivityActivity) getActivity()).getAddressLocation();
         //Zoom on the location that the user set, or on his GPS position if none found
         Log.d("MiniMapFragment TAG", "user current location is " + currentLocation.toString());
         if (location != null) Log.d("MiniMapFragmentTAG", "not null location us " + location.toString());
         if (location == null) location = new LatLng(currentLocation.getLatitude(), currentLocation.getLongitude());
         googleMap.animateCamera(CameraUpdateFactory.newLatLng(location));
-        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(location, 12.0f));
+        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(location, ZOOM_VALUE));
```

Comment:
Yes, clean to import this ZOOM_VALUE constant from the MainMapFragment 👍

Reply:
yes, I followed Kepler's advice on this one

Reply:
Yes, I remember! 👌